aws_lambda_powertools/utilities/parameters/secrets.py

```diff
@@ -60,14 +62,22 @@ class SecretsProvider(BaseProvider):
 
     client: Any = None
 
-    def __init__(self, config: Optional[Config] = None, boto3_session: Optional[boto3.session.Session] = None):
+    def __init__(
+        self,
+        config: Optional[Config] = None,
+        boto3_session: Optional[boto3.session.Session] = None,
+        boto3_client: Optional[boto3.client] = None,
```

Comment:
Same as the SSM comment: https://youtype.github.io/boto3_stubs_docs/mypy_boto3_secretsmanager/#secretsmanagerclient

Reply:
fixed